tests_utils/test-cn-m3-Docker.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+#
+# Test script for the following scenario
+# - Docker container named 'gateway_test' attached to a standalone M3 OpenNode and standalone M3 ControlNode
+# - The ControlNode board's FTDI eeprom has been set with the name ControlNode (using the IotLab ftdi-eeprom-config tool)
+
+# Experiment profile (sniffer) and firmware
+profile="../gateway_code/tests/profiles/radio_sniffer_3.json"
+firmware="../gateway_code/static/m3_autotest.elf"
+
+# Mocked user workspace and experiment folders (usually mounted by nfs)
+# here, they are created in the (alreadyà running container
+user="test"
```

Comment:
you should use or modify one of the makefile targets (I think `make run` will do what you need, just add a -v mapping a folder in /tmp to /iotlab/users/.../experiment)